docs/components/DataList/Web.stories.tsx

```diff
@@ -254,9 +268,17 @@ const Template: ComponentStory<typeof DataList> = args => {
 
   function getLoadingState() {
     if (loadingInitialContent) return "initial";
-    if (loadingNextPage) return "loadingMore";
+    if (loadingNextPage || loadingIDs) return "loadingMore";
     return args.loadingState;
   }
+
+  async function handleSelectAll() {
+    if (totalCount === selected.length) return setSelected([]);
+
+    const idsQuery = await getIDs();
+    const ids = idsQuery?.data?.allPeople?.edges?.map(({ node }) => node.id);
```

Comment:
All the `?` jumped out 😆 

Is it too much to split up to avoid nesting?

```
    const edges = idsQuery?.data?.allPeople?.edges;
    if (!edges) return;

    const ids = edges.map(({ node }) => node.id);

Reply:
it's still gonna return undefined tho haha

Reply:
😆 ok truee

Reply:
hmm now that I think of it, we promoting good code so imma do that suggestion.